www/src/models/repo.ts

```diff
@@ -16,6 +22,9 @@ export const RepoFragment = gql`
     description
     releaseStatus
     documentation
+    docs {
```

Comment:
Is this going to be sideloaded in repo list queries?  It's far too heavy to be done in those contexts if so

Reply:
Okay, I'll move to separate query for app pages only.